package.json

```diff
@@ -63,7 +66,9 @@
 	"jest": {
 		"collectCoverageFrom": [
 			"src/**/*.ts",
-			"!src/index.ts"
+			"!src/tmp/**/*.ts",
```

Comment:
I don't want my `tmp` folder to be included in code coverage, for obvious reasons.